7_deploy_osh/roles/suse-osh-setup/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Please remove this part of the change too (see https://github.com/SUSE-Cloud/socok8s/pull/93)